RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -22,7 +23,20 @@
 eras.trackingPhase1.toModify(detachedQuadStepSeedLayers,
     layerList = RecoPixelVertexing.PixelTriplets.quadrupletseedmerging_cff.PixelSeedMergerQuadruplets.layerList.value()
 )
-
+_layerListForPhase2 = ['BPix1+BPix2+BPix3', 'BPix2+BPix3+BPix4',
+                       'BPix2+BPix3+FPix1_pos', 'BPix2+BPix3+FPix1_neg',
+                       'BPix1+BPix2+FPix1_pos', 'BPix1+BPix2+FPix1_neg',
+                       'BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg',
+                       'BPix1+FPix1_pos+FPix2_pos', 'BPix1+FPix1_neg+FPix2_neg',
+                       'FPix1_pos+FPix2_pos+FPix3_pos', 'FPix1_neg+FPix2_neg+FPix3_neg',
+                       'FPix2_pos+FPix3_pos+FPix4_pos', 'FPix2_neg+FPix3_neg+FPix4_neg',
+                       'FPix3_pos+FPix4_pos+FPix5_pos', 'FPix3_neg+FPix4_neg+FPix5_neg',
+                       'FPix4_pos+FPix5_pos+FPix6_pos', 'FPix4_neg+FPix5_neg+FPix6_neg',
+                       'FPix5_pos+FPix6_pos+FPix7_pos', 'FPix5_neg+FPix6_neg+FPix7_neg',
+                       'FPix6_pos+FPix7_pos+FPix8_pos', 'FPix6_neg+FPix7_neg+FPix8_neg',
+                       'FPix6_pos+FPix7_pos+FPix9_pos', 'FPix6_neg+FPix7_neg+FPix9_neg'
+]
+eras.trackingPhase2PU140.toModify(detachedQuadStepSeedLayers, layerList = _layerListForPhase2)
```

Comment:
Ah, I missed the one-line difference. Then I'd suggest to keep the explicit list in DetachedQuadStep with a comment that there is a bug that will be fixed later. (validation and integration will be much easier if there are no relevant changes in the expanded configuration and no changes in DQM plots).